src/gsi/convinfo.f90

```diff
@@ -242,9 +244,9 @@ subroutine convinfo_read
 
     do i=1,nlines
        cflg=' '
-       iotype='       '
+       iotype='        '
        read(lunin,1030)cflg,iotype,crecord
-       if (cflg==' '.and.iotype=='       ') then
+       if (cflg=='  '.and.iotype=='        ') then
```

Comment:
Any reasons to add a new space to compare with for cflg?

Reply:
As noted in the documentation section of convinfo_read, iotype was extended to 8 characters-long to allow for the character length of the new observation type **gnssrspd**.

Reply:
Was  the cflg extended to 2?

Reply:
Three questions about the observation type assignment
(1) Based on the code, both GNSS-R L2 and CYGNSS wind speed at 10 meters, correct?
(2) Are both (GNSS-R and CYGNSS)  assigned to GSI obstype as 298?
(3) Could you post the entries (parameters) for `gnssrspd` in convinfo here for our reference?

Reply:
@ xincjin-NOAA: thanks for your comment. The extra space in "cflg==" was a typo of merge conflict. convinfo.f90 has been updated. 

@emilyhcliu, here are the answers to your questions:

(1) Based on the code, both GNSS-R L2 and CYGNSS wind speed at 10 meters, correct? Yes
(2) Are both (GNSS-R and CYGNSS) assigned to GSI obstype as 298? Yes
(3) Could you post the entries (parameters) for gnssrspd in convinfo here for our reference?

![Screen Shot 2024-05-20 at 11 32 59 AM](https://github.com/NOAA-EMC/GSI/assets/104657787/82784130-6fd9-4d83-ba0d-cad9b07e65ee)